src/DataFixtures/ReservationsFixtures.php

```diff
@@ -0,0 +1,331 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Reservation;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+
+class ReservationsFixtures extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        $set1 = $this->collectData(
+            new \DateTime('2018-11-27'),
+            '20',
+            new \DateTime('2018-11-29'),
+            '20',
+            'Zigmas',
```

Comment:
nepatinka sitas kodas visiskai. 
1) neturetu vardu imt is uzeriu fixtures?
2) datos turetu but dinamiskos. kas jeigu paleisi fixtures ir dienos bus praeity?